src/lib.rs

```diff
@@ -15,109 +15,5 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 extern crate alloc;
```

Comment:
I think we can remove the library and only make `ctap2` a binary.

Reply:
We need the library for now for our examples.